mods/d2k/rules/vehicles.yaml

```diff
@@ -442,7 +442,7 @@ deviator:
 		BuildPaletteOrder: 40
 		BuildDuration: 373
 		BuildDurationModifier: 40
-		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry, Aircraft\n \n  Atreides:      +Range\n  Harkonnen:  +Health\n  Ordos:          +Speed
+		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry
```

Comment:
> Removed Combat Tank desc's Side Specific info. It is no longer true.

This side specific info still valid but maybe not up to date.
Atreides: I dont find anything about the improved range, their gun has the same [range](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/weapons/largeguns.yaml#L40) as the others
Harkonnen: [Health](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/rules/vehicles.yaml#L498)
Ordos: [Speed](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/rules/vehicles.yaml#L511)